pages/blueprint/index.js

```diff
@@ -285,15 +237,8 @@ class BlueprintPage extends React.Component {
                               data-toggle="dropdown"
                               aria-haspopup="true"
                               aria-expanded="false"
-                            >
-                              Commit 5<span className="caret" />
-                            </button>
+                            ></button>
```

Comment:
Similar to previous comment, I think we could remove this <button> altogether until the functionality for this implemented.